content/projects/remoting/index.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: project
+title: "Jenkins Remoting"
+section: projects
+tags:
+- remoting
+- agent
+- agents
+- slave
+- slaves
```

Comment:
It is to aggregate links to old blogposts. But I agree it rather makes sense to update labels there